CHANGELOG.md

```diff
@@ -1,5 +1,96 @@
 # Change Log
 
+## v0.10.0 2021-02-01
+
+[full changelog](https://github.com/executablebooks/jupyter-book/compare/v0.9.1...d3c78097edda4fefb672e32344c3806c9cdc7a72) | [GitHub contributors page for this release](https://github.com/executablebooks/jupyter-book/graphs/contributors?from=2020-12-22&to=2021-01-30&type=c)
+
+This update focuses on new syntax features for MyST markdown, as well as a new configuration to enable MyST extensions. See below for more details.
+
+### New
+
+**MyST Parser version 0.13.x**
```

Comment:
You don't really need to specifically make these definition headers bold, they already "wrapped" in `<dt>` tags and so the theme will take care of them